### ISSUE TITLE
Using user SublimeLinter.sublime-settings in addition to (or instead of) .jshintrc

### ISSUE THREAD

**Issue by eugenesvk** [SublimeLinter/SublimeLinter-jshint#88]
I'm using ST in at least 4 different environments: Win, OSX, Ubuntu and portable from a USB stick. I'd like to have the same config files everywhere, including configs for jshint linter. The way jshint is looking for `.jshintrc` file isn't great as it's not looking beyond $HOME and I've moved all my configs to `$Home/.config` to avoid pollution. However, even if this weren't so, this still wouldn't be ideal as my USB portable ST wouldn't be able to see it.

Now, as far as I understand (also from https://github.com/SublimeLinter/SublimeLinter-jshint/issues/8) it's not possible to use User's `SublimeLinter.sublime-settings` file to store settings (they're simply ignored since you moved to reading settings from  `.jshintrc`), which makes my task of simple settings sync impossible.

Would you please allow to have all settings stored in the `SublimeLinter.sublime-settings` file in the ST user folder and prioritize these options over `$HOME/.jshintrc`?

Basically using this hierarchy from http://sublimelinter.readthedocs.org/en/latest/settings.html with an added `$HOME/.jshintrc` that should be below User settings (so that a portable ST wouldn't be overwritten by a host machine config file).

```
Inline overrides
Inline settings
.sublimelinterrc settings
Project settings
User settings
Default settings
```

P.S.
Alternatively, is there a way to set a relative path (to the User settings folder, then at least I could put  `.jshintrc` file there and link to it and this link would work regardless of the system I'm running ST on, portable or otherwise)?

**Comment by mokkabonna** [comment]
Pr welcome, not something I will prioritize.
